versions-maven-plugin/src/test/java/org/codehaus/mojo/versions/UpdateParentMojoTest.java

```diff
@@ -204,6 +204,7 @@ public void testParentDowngradeForbiddenWithRange()
         assertThat(newVersion, nullValue());
     }
 
+    /*
```

Comment:
Why disable this test?

In such case I prefer to delete code instead of commented out. 

When we want to left commented code we should add information why is commented, what is plan to restore, etc ...